tensorflow/lite/micro/tools/ci_build/test_project_generation.sh

```diff
@@ -49,9 +49,10 @@ popd > /dev/null
 
 rm -rf ${TEST_OUTPUT_DIR}
 
-# Check that we can export a TFLM tree with additional makefile options.
-TEST_OUTPUT_DIR_CMSIS=$(mktemp -d)
+# Remove existing state prior to testing project generation for cortex-m target.
```

Comment:
nit: is this only for cortex-m?

Reply:
for now, yes. We are not testing any other target platform with the project generation script.